.gitignore

```diff
@@ -213,3 +214,6 @@ Network Trash Folder
 Temporary Items
 .apdisk
 
+#Netbeans
+.nb-gradle/
```

Comment:
You can add full ignore list for NetBeans by adding content generated by [gitignore.io](https://www.gitignore.io/):
https://www.gitignore.io/api/netbeans